CRM/Core/PrevNextCache/Sql.php

```diff
@@ -97,37 +97,37 @@ public function markSelection($cacheKey, $action, $cIds = NULL) {
       if (is_array($cIds)) {
         $cIdFilter = "(" . implode(',', $cIds) . ")";
         $whereClause = "
-WHERE cacheKey LIKE %1
```

Comment:
I confirmed that where the cacheKey is passed to this function it is always qfKey from the url - which is either the qfKey or 'civicrm search ' + $('input[name=qfKey]'